src/jazayeri_lab_to_nwb/watters/watterstrialsinterface.py

```diff
@@ -61,6 +61,9 @@ def _read_file(self, file_path: FolderPathType):
         for i in range(n_trials):
             # get trial start time
             start_time = data_dict["task/trials.start_times.json"][i]
+            if np.isnan(start_time):
+                print(f"Start time for trial {i} is NaN. Dropping this trial.")
```

Comment:
I thought this was only in the case of the very first trial, right? Not any other trial

I'd also use a `warning.warn` so the output goes to `sys.stderr` instead of `stdout`

Reply:
good point, I can make the condition stricter to throw an error for NaNs if it isn't the first trial